OpenRA.Mods.Common/Traits/Modifiers/DisabledOverlay.cs

```diff
@@ -15,13 +15,19 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Use together with CanPowerDown/RequiresPower on buildings or Husk for vehicles.")]
-	public class DisabledOverlayInfo : TraitInfo<DisabledOverlay> { }
+	public class DisabledOverlayInfo : UpgradableTraitInfo
```

Comment:
We already have `UpgradeOverlay` trait for doing this with upgrades.  This trait is a legacy/deprecated leftover for working with the old `IsDisabled` code (which IMO may need to be cleaned up before `INotifyBuildComplete`).